### DISCUSSION TITLE
Icon Color for Filament\Actions\StaticAction

### DISCUSSION THREAD

**Discussion by Carnicero90** [filamentphp/filament#11812] [Ideas]
@danharrin may I ask if there is any strong reason not to use HasIconColor in Filament\Actions\StaticAction? Cos (unless I'm missing something) I dont think I can customize icon color in a table action. If not, I'm eager to submit a pr to implement it.
<img width="284" alt="image" src="https://github.com/filamentphp/filament/assets/78483736/52254a01-aae9-44cc-bc4b-9c42b1ef7528">

**Comment by Carnicero90** [comment]
I mean *just* the icon color, keeping the default one for the action label

**Comment by danharrin** [comment]
Sounds good, then you'd need to pass it through to packages/actions/resources/views/grouped-action.blade.php and packages/actions/resources/views/grouped-group.blade.php (if you add the trait to ActionGrouo too)

**Reply by Carnicero90** [reply]
Indeed, that's right where I need it (currently I am kinda "patching" it in my app via extra attributes + css hooks).
